forte/genci/string_lists_makers.cc

```diff
@@ -387,6 +387,97 @@ H3List make_3h_list(const StringList& strings, std::shared_ptr<StringAddress> ad
     return list;
 }
 
+H4List make_4h_list(const StringList& strings, std::shared_ptr<StringAddress> addresser,
```

Comment:
Can you time this for say, a CAS(10,10) calculation (without computing the 4-RDM) and report it in the PR notes?